CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHeartRateSampleProcessor.swift

```diff
@@ -458,12 +463,17 @@ internal class CRFHeartRateSampleProcessor {
         // it is reproduced here for testing purposes.
         let drop = dropSeconds > 0 ? dropSeconds * samplingRate - 1 : 0
         let x = input.dropFirst(drop).map({ $0.isFinite ? $0 : 0 })
+        // get minimum value in the input window and offset each signale frame by the minimum value observed in the window
+        let minValue = x.min()
+        let minnedOutX = x.map({$0 - minValue!})
```

Comment:
Do we need to worry about the forced unwrap of minValue?  Should it be:
`let minValue = x.min() ?? 0.0`
`let minnedOutX = x.map({$0 - minValue})`

Reply:
Not really, though it has less code smell. :)